endpoints-support.md

```diff
@@ -17,7 +17,7 @@
 <br /> 
  
 ## HPE OneView 
-|  Endpoints  |  Verb  |  v4400  |  v4600  |  v4800  |  v5000  |  v5200  | v5400               |
+|  Endpoints  |  Verb  |  v4400  |  v4600  |  v4800  |  v5000  |  v5200  | v5400               | v5600               |
```

Comment:
Table is not proper. Missing :----------------------: |. Fix this before submitting

Reply:
Made the reqd. change